src/Pages/Order/Payment/Payment.js

```diff
@@ -121,7 +121,7 @@ export default class Payment extends Component {
           </div>
           <div className="agreeBtnContainer">
             <div className="wrapAgreeCheckBox">
-              <label className="wrapAgreeCheckBox" id="finalCheckBox">
+              <label className="wrapAgreeCheckBox" for="finalCheckBox">
```

Comment:
```suggestion
              <label className="wrapAgreeCheckBox">
```
- for 없어도 될 것 같습니다
- 만약 쓴다면 htmlFor를 써야 합니다

Reply:
아하 알겠습니다!